src/test/functional/java/uk/gov/hmcts/reform/sandl/snlrules/functional/JwtTokenHelper.java

```diff
@@ -18,7 +18,7 @@ public static HttpHeaders createRulesAuthenticationHeader() {
         HttpHeaders headers = new HttpHeaders();
         headers.add(S2SAuthenticationService.HEADER_NAME,
             S2SAuthenticationService.HEADER_CONTENT_PREFIX
-                + createToken("nieWiedzialaJak", 5000, "snl-events")
+                + createToken("nieWiedzialaJak", 15000, "snl-events")
```

Comment:
can this token be taken from application.yml, instead of hardcoding here. ${management. management. rules. jwtSecret}